contrib/chinese/config.el

```diff
@@ -0,0 +1,32 @@
+;;; config.el --- Colors Layer configuration File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; Variables
+
+(defvar chinese-im-enable-pinyin nil
+  "Use Pinyin input method.")
+
+(defvar chinese-im-enable-wubi nil
+  "Use Wubi input method.")
```

Comment:
Can both of these input method variable be enabled at the same time ?
If the expected behavior is to enable either one or the other then we can unify the variables under one variable `chinese-default-input-method` with the possible values `pinyin`and `wubi`.

In any case I would make the names more obvious with `input-method` instead of  `im` and `enable` should follow the layer name. It gives:
- `chinese-enable-pinyin-input-method`
- `chinese-enable-wubi-input-method`